rails/spec/models/place_spec.rb

```diff
@@ -6,6 +6,118 @@
     it { should have_many :interview_stories }
   end
 
+  let(:community) { create(:community) }
+  let(:place) { described_class.new }
+
+  describe "#csv_headers" do
+    it "defines importable CSV headers" do
+      expect(described_class.csv_headers).to eq(
+        %i[
+          name
+          type_of_place
+          lat
+          long
+          region
+          description
+          photo
+          name_audio
+        ]
+      )
+    end
+  end
+
+  describe "#import" do
+    let(:mapped_headers) {{
+      name: "name",
+      type_of_place: "type_of_place",
+      description: "description",
+      region: "region",
+      long: "long",
+      lat: "lat"
+    }}
+
+    it "raises HeaderMismatchError when mapped headers are missing" do
+      expect {
+        described_class.import(
+          file_fixture('place_without_media.csv'),
+          community.id,
+          mapped_headers.merge({photo: "other"})
+        )
+      }.to raise_error(Importable::FileImporter::HeaderMismatchError)
```

Comment:
maybe a future improvement here is to add which headers were mismatched to the exception message?